### PR TITLE
Converted Nodes to sync strings (log entries) and Updated README 

### PR DESCRIPTION
**Pull request by thezhe**
The code is fairly simple now. It should be easy to build upon it. Feel free to add code for the logs to the two .cc files.

### CODE REVIEW

**Review by zhou-1 — COMMENTED** [review]
Nice work! Shall we also add in hash function for string?

File: src/CPISync/GossipNode.h [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+{
+class GossipNode
+{
+public:
+//Metadata
+string name;
+vector <string> log;
+vector <string> neighbors;
+//Data
+DB* db = nullptr;
```
Comment (wfystx) [inline src/CPISync/GossipNode.h:19]: I'll import the kvstore part code later

**Review by wfystx — APPROVED** [review]
Well done

File: src/CPISync/GossipNode.h (hunk 1 above)
Comment (thezhe) [reply src/CPISync/GossipNode.h:19]: Didn't get a chance to add the kvengine, so feel free to do that.

**Pull request merged**